modules/ROOT/pages/configure-message-flows.adoc

```diff
@@ -93,13 +66,43 @@ This is the target message type and the Endpoint where you want to route the tra
 
 * xref:endpoint-https-send.adoc[HTTP and HTTPS Send Configuration]
 
-. After you create the source, map, and target sections in the interface, confirm that each shows a green check mark next to it in the flow), and click *Deploy* to deploy the configurations to your Mule Runtime.
+. After you create the source, map, and target sections in the interface, confirm that each shows a green check mark next to it in the flow, and click *Deploy* to deploy the configurations to your Mule Runtime.
 +
-Your message flow configuration is deployed to your Mule runtime engine and you can start receiving messages from your partner. Use the Activity page to track processed messages.
+. Your message flow configuration is deployed to your Mule runtime engine and you can start receiving messages from your partner.
 +
-Once a message flow is successfully deployed, you can track its processed messages from the Activity page.
+. Once a message flow is successfully deployed, you can track its processed messages from the Activity page.
+
+== Test Message Flows
+
+There are a couple of reasons why you should send test messages through the message flow after you deploy:
+
+* Ensure that you are receiving messages and any acknowledgments from your partners.
+
+* Make sure that the DataWeave map is working and your backend systems receive the transformed message.
+
+You can test the flow by deploying it in a sandbox environment, then undeploying it, and finally deploying it to a production environment.
+
+Test your message flow in your environment:
+
+. Initiate a transmission on the receiving endpoint.
+
+. From within Anypoint Partner Manager select *Activity* and select the transmission from the list.
+
+. Scan the Transmission Flow sections for an exclamation point (!) in any of the rectangle boxes next to the displayed flow stages as this indicates there is an issue with the transmission.
+
+. If view payloads are enabled, review the received payload by clicking *View payload* next to the Received transmission section.
```

Comment:
The previous step indicated the possibility of issues with the transmission; does this step address that?

Reply:
I'm not sure that "view payloads are enabled" is grammatical: a type of payload, or a control called View Payloads?
Clarify...or consider this:
```suggestion
. Review the received payload by clicking *View payload*.
```
Review it for what?
Also, do we need to add a subordinate para that explains what you mean by "if view payloads are enabled"? Are we saying that the controls in this step might not appear in the UI if this has not been enabled?

Reply:
Fixed.